flask_common/test_helpers.py

```diff
@@ -13,6 +14,10 @@ def __init__(self, members):
     def __eq__(self, other):
         return set(other) == set(self.members)
 
+    def __ne__(self, other):
+        return not self.__eq__(other)
```

Comment:
Should this be `return not self == other` instead? See https://stackoverflow.com/a/30676267/1579058